pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerBase.java

```diff
@@ -73,6 +81,14 @@ public MessageId send(T message) throws PulsarClientException {
         return new TypedMessageBuilderImpl<>(this, schema);
     }
 
+    // TODO: same parameterized type `T` with producer for the moment,
```

Comment:
Create a github issue for this change and add the github issue in this TODO comment? It is a good practice to always track TODO items in github issues.

Reply:
Should I just edit the master issue, with a checklist, or new a issue for each TODO?